modules/wpml/wpml-config.php

```diff
@@ -594,4 +578,140 @@ private function get_field_attribute( SimpleXMLElement $field, $attribute_name )
 
 		return trim( (string) $attributes[ $attribute_name ] );
 	}
+
+	/**
+	 * Returns all wpml-config.xml files in MU plugins.
+	 *
+	 * @since 3.3
+	 *
+	 * @return string[] A context identifier as array key, a file path as array value.
+	 *
+	 * @phpstan-return array<string, string>
+	 */
+	private function get_mu_plugin_files() {
+		if ( ! is_dir( WPMU_PLUGIN_DIR ) ) {
+			return array();
+		}
+
+		$files = array();
+
+		// Search for top level wpml-config.xml file.
+		$file_path = WPMU_PLUGIN_DIR . '/wpml-config.xml';
+
+		if ( file_exists( $file_path ) ) {
+			$files['mu-plugins'] = $file_path;
+		}
+
+		// Search in proxy loaded MU plugins.
+		try {
```

Comment:
`try` and `catch` are here because of `DirectoryIterator::__construc()`?
If so it shouldn't raise any exception or error since `WPMU_PLUGIN_DIR` is checked earlier, should it?

Reply:
You're right, I added this before the `is_dir()` check, it has no value now.
Removed in [47eca19](https://github.com/polylang/polylang/pull/1140/commits/47eca19d2fb23ce3fb795784611725dcb1199ad1).